docs/details/commands/pmm-admin.md

```diff
@@ -71,18 +76,23 @@ PMM communicates with the PMM Server via a PMM agent process.
 ### INFORMATION COMMANDS
 
 `pmm-admin list --server-url=server-url [FLAGS]`
-:    Show Services and Agents running on this Node.
+:    Show Services and Agents running on this Node, and mode of vmagent (push/pull).
```

Comment:
"and information on how the agent operates with data(push/pull)"
Let's avoid vmagent here because it's  Agent related information and vmagent just helps to do this